server/src/main/java/net/cryptic_game/backend/server/server/websocket/endpoints/WebSocketUserEndpoints.java

```diff
@@ -183,4 +183,21 @@ public ApiResponse delete(final ApiClient client, @ApiParameter("password") fina
 
         return new ApiResponse(ApiResponseType.OK);
     }
+
+    @ApiEndpoint("get")
+    public ApiResponse get(final ApiClient client, @ApiParameter("uuid") final UUID userId) {
+        Session session = client.get(Session.class);
+        if (session == null || !session.isValid()) {
+            return new ApiResponse(ApiResponseType.FORBIDDEN, "NOT_LOGGED_IN");
+        }
+
+        final User user = User.getById(userId);
+
+        if (user == null) {
+            return new ApiResponse(ApiResponseType.NOT_FOUND, "USER_NOT_FOUND");
+        }
+
+        return new ApiResponse(ApiResponseType.OK, JsonBuilder.create("username", user.getName())
```

Comment:
Rename `username` to `name`, `last_seen` to `last` and add `id`.

Reply:
Sezalisize the user by submitting the user to JsonBuilder without any other actions that replaces the idear of @JannikEmmerich